ghcide.cabal

```diff
@@ -355,7 +355,7 @@ test-suite ghcide-tests
         text
     hs-source-dirs: test/cabal test/exe test/src bench/lib
     include-dirs: include
-    ghc-options: -threaded -Wall -Wno-name-shadowing
```

Comment:
You are also removing `-threaded`. Do we not run the tests in parallel? Or at least have the option to do so? Or does this actually break our test suite.

Reply:
We don't run the tests in parallel, see:

https://github.com/haskell/ghcide/blob/114e184e3228afbf20c3a3ca57e2ebb8771cce1d/test/exe/Main.hs#L60-L66